docs/instrumentation/tornado/starlette/tornado.rst

```diff
@@ -0,0 +1,7 @@
+OpenTelemetry Tornado Instrumentation
```

Comment:
Is this the right place for this doc? i'm confused about the `tornado/starlette/tornado.rst` path.

Reply:
Sorry about that. No idea how that happened. My guess is an unintentional drag in vscode sidebar :)